remotespark/magics/sparkmagicsbase.py

```diff
@@ -47,7 +47,7 @@ def _sqlquery(cell, samplemethod, maxrows, samplefraction):
 
     @staticmethod
     def print_endpoint_info(info_sessions):
-        sessions_info = ["        {}".format(i) for i in info_sessions]
+        sessions_info = ["        {}\n".format(i) for i in info_sessions]
```

Comment:
It seems we need to reformat to an HTML table or similar, as this has gotten messy already? What do you think?